flypipe/config.py

```diff
@@ -15,6 +15,7 @@ class _Config:
         "require_node_description": False,
         "require_schema_description": False,
         "default_run_mode": RunMode.SEQUENTIAL.value,
+        "node_run_max_workers": 4,

```

Comment:
I think if would be better if we leave:
`os.cpu_count() - 1`

Also can please update documentation `docs/source/configuration/configuration.md`?

Reply:
As per discussions, changed to os.cpu_count(), updated documentation and added a note to warn about limited parallelization on Python CPU bound threads